src/measure_theory/bochner_integration.lean

```diff
@@ -262,7 +262,7 @@ begin
   rcases mem_range.1 hp with ⟨a, rfl⟩,
   by_cases eq : f a = g a,
   { dsimp only [pair_apply], rw eq },
-  { have : μ ((pair f g) ⁻¹' {(f a, g a)}) = 0,
+  { have : μ ((pair f g) ⁻¹' {prod.mk (f a) (g a)}) = 0,
```

Comment:
Could we use `\<\>` instead of `()` or `prod.mk`?

Reply:
When I `leanproject up` to get oleans for this branch, I need to do a full recompile anyway. Not sure why.

Reply:
3.18.4 will be out today, then we can revert this hack as well.

Reply:
Yep, done!